elan/rpc/gc.go

```diff
@@ -83,9 +83,9 @@ func (s *server) deleteAll(ctx context.Context, blobs []*ppb.Blob, hard bool) er
 	var me *multierror.Error
 	for _, blob := range blobs {
 		key := blob.CachePrefix + blob.Hash
-		log.Notice("Deleting blob %s", key)
 		if exists, err := s.bucket.Exists(ctx, key); exists {
 			if err := s.bucket.Delete(ctx, key, hard); err != nil {
+				log.Error("Error deleting blob: %v/home/hpitkeathly/.local/state/nvim/swap//%home%hpitkeathly%please-servers%elan%rpc%rpc.go.swp", err)
```

Comment:
this looks like a copy&paste typo, but also might be something important, i have no idea

Reply:
:aha: You're right that is a c&p error. That's the location of a previous vim swp file I needed to remove before I could open a new one.